src/client.ts

```diff
@@ -55,11 +56,13 @@ export default class Client {
 
   constructor(sdkKey: string, options: Options = {}) {
     this.sdkKey = sdkKey;
+    this.options = { ...defaultOptions, ...options };
```

Comment:
I prefer to do validation on input arg in this case `options`

Reply:
The validation that was done before is still happening (on `pollInterval`), but with this, I'm just munging the custom input `options` into the `defaultOptions`, so we have a complete set to pass into the child classes.

Is there a better way to do that?